layer1/transaction/backend.go

```diff
@@ -177,7 +177,7 @@ func newSharesReceipt() *SharedReceipt {
 	return &SharedReceipt{doneChan: make(chan struct{})}
 }
 
-// Function to check if a receipt is ready
+// IsReady is a function to check if a receipt is ready
```

Comment:
Function is implied.

```suggestion
// IsReady to check if a receipt is ready.
```